cypress/TestCases/FireboltCertification/SDKBehavior/LocalizationSDK.feature

```diff
@@ -6,16 +6,22 @@ Feature: Localization_SDK
         And 3rd party 'certification' app is launched
 
     @sdk
-    Scenario Outline: Localization.<SetMethod> - Positive Scenario: <Scenario> with undefined params
-        When 1st party app invokes the 'Firebolt' API to '<API_Key>'
-        Then 'Firebolt' platform responds to '1st party app' with '<Validation_Key>'
+    Scenario Outline: Localization.<Scenario> - Positive Scenario: <Attribute> with undefined params
+        Given we test the 'LOCALIZATION_SDK_BEHAVIOUR' getters and setters '<Attribute>'
+        When 1st party app invokes the 'Firebolt' API to set value
+        Then 'Firebolt' platform responds to '1st party app' set API
 
         Examples:
-            | Scenario                | SetMethod                  | API_Key                                    | Validation_Key                                |
-            | locality                | setLocality                | set locality without params                | expected localization locality                |
-            | postalCode              | setPostalCode              | set postalcode without params              | expected localization postalcode              |
-            | countryCode             | setCountryCode             | set countrycode without params             | expected manage localization countrycode      |
-            | language                | setLanguage                | set language without params                | expected manage localization language         |
-            | preferredAudioLanguages | setPreferredAudioLanguages | set preferredaudioLanguages without params | expected localization preferredaudiolanguages |
-            | locale                  | setLocale                  | set locale without params                  | expected manage localization locale           |
-            | timeZone                | setTimeZone                | set timezone without params                | expected localization timezone                |
+            | Scenario       | Attribute   |
+            | setLocality    | locality    |
+            | setPostalCode  | postalCode  |
+            | setCountryCode | countryCode |
+            | setLanguage    | language    |
+            | setLocale      | locale      |
```

Comment:
we need below right 

 Scenario Outline: Localization.<Scenario> - Positive Scenario: <Attribute> with undefined params
        Given we test the 'LOCALIZATION_SDK_BEHAVIOUR' getters and setters '<Attribute>'
       When 1st party app invokes the 'Firebolt' API to set value to '<attribute>'
        When 1st party app invokes the 'Firebolt' API to set value
        Then 'Firebolt' platform responds to '1st party app' set API


That way we would know what is the value that we set and what should be the content validation for last test step

Reply:
also where are the other test cases ?

Reply:
Yes, As part of this ticket we covered one testcase and remaning will be taken in another ticket